docs/infra/deployment.md

```
@@ -0,0 +1,139 @@
+database_ip=db
+database_port=3306
+database_user=root
+database_password=your_secure_password
+database_schema=tanjun
+```
+
+#### Health Checks
+
+The bot includes built-in health checks accessible at:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)